docs/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1819 @@
+{
```

Comment:
<p>I'd include an example that actually works. Otherwise I wouldn't even expect it to work because it's not a RV directly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>

Reply:
<p>Any particular suggestion on an interesting example?</p>

Reply:
I'm actually surprised it doesn't work with `cumsum()`.  @ricardoV94 would know.